src/draw/vg_lite/lv_draw_vg_lite.c

```diff
@@ -112,36 +117,49 @@ static void draw_execute(lv_draw_vg_lite_unit_t * u)
     vg_lite_identity(&u->global_matrix);
     vg_lite_translate(-layer->buf_area.x1, -layer->buf_area.y1, &u->global_matrix);
 
+    LV_LOG_USER("draw task: %d", t->type);
+
+    bool has_flushed = true;
+
     switch(t->type) {
         case LV_DRAW_TASK_TYPE_LABEL:
             lv_draw_vg_lite_label(draw_unit, t->draw_dsc, &t->area);
```

Comment:
Every other task type calls `lv_vg_lite_flush(u)`, but label not.
Is it a special case? (I'm not familiar with VG Lite)